src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -439,4 +461,14 @@ export default {
 	background-color: var(--color-background-hover);
 	border-radius: 8px;
 }
+
+.highlight-animation {
```

Comment:
missing the border-radius compared to the hover?

Reply:
I didn't intend to add one, but you're right that it might look better.
I'll add it

Reply:
Added